src/frequenz/sdk/timeseries/_moving_window.py

```diff
@@ -46,6 +47,13 @@ class MovingWindow:
     [`OrderedRingBuffer`][frequenz.sdk.timeseries._ringbuffer.OrderedRingBuffer]
     documentation.
 
+    Resampling might be required to reduce the number of samples to store
+    without losing precision, and it can be set by specifying the resampler
```

Comment:
I'm not sure what you want to say without loosing precision. We are loosing precision for when we downsample but that's acceptable in many cases.

Reply:
I meant that we can reduce the number of samples without loosing *much* precision through resampling. I'd then remove `without loosing precision` as it is confusing and it isn't generic in the sense it only applies for downsampling where it is somehow implicit an acceptable loss of precision